plugin-bazel/src/main/kotlin/org/jetbrains/bazel/symbols/BazelTargetSymbolPointer.kt

```
@@ -0,0 +1,46 @@
+) : Pointer<BazelTargetSymbol> {
+
+  override fun dereference(): BazelTargetSymbol? {
+    return try {
+      val label = Label.parse(labelString)
+      BazelTargetSymbol(
+        label = label,
+        buildFilePath = buildFilePath,
+        targetType = targetType,
+        aliases = aliases
+      )
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)